integration_tests/src/wallet_process.rs

```diff
@@ -226,6 +226,7 @@ pub fn get_default_cli() -> Cli {
         profile_with_tokio_console: false,
         view_private_key: None,
         spend_key: None,
+        reachability: Some(ReachabilityMode::Private),
```

Comment:
Maybe we need to be more aggressive in establishing a connection for the integration tests? I see many of them are failing.
```suggestion
        reachability: Some(ReachabilityMode::Auto),
```

Reply:
Integration tests do not have a public IP so Private is appropriate